frontend-react/src/components/Admin/EditSenderSettings.test.tsx

```diff
@@ -89,8 +93,52 @@ describe("EditSenderSettings", () => {
         });
 
         expect(processingTypeField).toHaveValue(testProcessingType);
-        fireEvent.click(screen.getByTestId("submit"));
+        fireEvent.click(editJsonAndSaveButton);
         fireEvent.click(screen.getByTestId("editCompareCancelButton"));
         fireEvent.click(screen.getByTestId("senderSettingDeleteButton"));
     });
+
+    describe("should validate name", () => {
+        const consoleTraceSpy = jest.fn();
+
+        beforeEach(() => {
+            jest.spyOn(console, "trace").mockImplementationOnce(
+                consoleTraceSpy
+            );
+        });
+
+        afterEach(() => {
+            jest.resetAllMocks();
+        });
+
+        test("should display an error if name value is prohibited", async () => {
```

Comment:
Minor point, but this doesn't need to be `async` since nothing's being `awaited` :)

Reply:
will remove